test/soca/gw/jjob_prep.sh

```diff
@@ -6,13 +6,25 @@ srcdir=$2
 # export env. var.
 source "${srcdir}/test/soca/gw/runtime_vars.sh" "${bindir}" "${srcdir}"
 
-OCNDIR="${ROTDIR}/${PSLOT}/gdas.${PDY}/${gcyc}/ocean/" 
+OCNDIR="${ROTDIR}/${PSLOT}/gdas.${PDY}/${gcyc}/ocean/"
 
 rm -r ${OCNDIR}
 
 # prepare background from previous cycle
 mkdir -p ${OCNDIR}
 cp -r "${bindir}/test/soca/bkg/"* ${OCNDIR}
 
+# detemine machine from config.base
+machine=$(echo `grep 'machine=' $EXPDIR/config.base | cut -d"=" -f2` | tr -d '"')
+
 # run prep jjob
-"${HOMEgfs}/jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP"
+if [[ ${machine} == 'CONTAINER' ]]; then
+    "${HOMEgfs}/jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP"
+else
+    sbatch --ntasks=1 \
```

Comment:
just a note here that we probably need to instead do this so that it is defining a 'submit command' somewhere and sourcing it so that this doesn't need repeated across multiple scripts.